open-craft/opencraft
# BB-2422 cleanup unused views

Clean up unused views in OCIM & link to Terms of Service, Privacy Policy on OpenCraft website

**Merge deadline**: None

**Testing instructions**:

1. Clone this branch
2. Run all tests with 'make test'
3. Open localhost:5000 and verify that you can still log in & log out
4. Verify that unauthenticated users get redirected to localhost:3000 where they can login, logout & register

**Reviewers**
